crates/oxc_minifier/src/peephole/replace_known_methods.rs

```diff
@@ -1619,27 +1651,34 @@ mod test {
     }
 
     #[test]
-    #[ignore]
     fn test_fold_math_functions_clz32() {
-        test("Math.clz32(0)", "32");
-        let mut x = 1;
-        for i in (0..=31).rev() {
-            test(&format!("{x}.leading_zeros()"), &i.to_string());
-            test(&format!("{}.leading_zeros()", 2 * x - 1), &i.to_string());
-            x *= 2;
-        }
-        test("Math.clz32('52')", "26");
-        test("Math.clz32([52])", "26");
-        test("Math.clz32([52, 53])", "32");
+        test_value("Math.clz32(0)", "32");
+        test_value("Math.clz32(0.0)", "32");
+        test_value("Math.clz32(-0.0)", "32");
+        // FIXME not implemented yet
+        // let mut x = 1;
+        // for i in (0..=31).rev() {
+        //     test(&format!("{x}.leading_zeros()"), &i.to_string());
+        //     test(&format!("{}.leading_zeros()", 2 * x - 1), &i.to_string());
+        //     x *= 2;
+        // }
```

Comment:
I think the ported code was incorrect. The first argument of `test` should include `Math.clz32`.
https://github.com/google/closure-compiler/blob/a79cf3ebeb1c8a26154701089c3c431194ac0b0a/test/com/google/javascript/jscomp/PeepholeReplaceKnownMethodsTest.java#L579-L583